packages/loader/container-utils/src/error.ts

```diff
@@ -194,9 +194,10 @@ export function wrapErrorAndLog<T extends IWriteableLoggingError>(
         ? newError.errorInstanceId
         : undefined;
 
-    logger.sendErrorEvent({
+    logger.sendTelemetryEvent({
```

Comment:
But this is common code and it will modify behavior at other places too.

Reply:
That's the intention :) This logging only to make sure that the data is not lost at all.
But the user of this object controls and determines if and how it will be logged. If caller believes it's not an error, it will not log it as such.